src/masonite/commands/RuleEnclosureCommand.py

```diff
@@ -0,0 +1,17 @@
+"""New Model Command."""
+
+from ..commands import BaseScaffoldCommand
+
+
+class RuleEnclosureCommand(BaseScaffoldCommand):
+    """
+    Creates a new rule enclosure.
+
+    rule:enclosure
+        {name : Name of the rule enclosure}
+    """
+
+    scaffold_name = 'Rule'
+    postfix = ""
```

Comment:
Should be:
```suggestion
    postfix = ''
```
Since all the other vairables are assinged with a single and not a dobule quote.